rviz_assimp_vendor/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ else()
   set(ASSIMP_CMAKE_FLAGS "-DCMAKE_INSTALL_LIBDIR=lib")
 
   set(ASSIMP_CXX_FLAGS "-std=c++14 ${CMAKE_CXX_FLAGS}")
+  set(ASSIMP_C_FLAGS "-Wno-deprecated-non-prototype ${CMAKE_C_FLAGS}")
```

Comment:
I'm on board with suppressing this warning, but I think we should have a comment saying why we did it (so we can possibly remove it later).  Something like:

```
# assimp version 5.3.1 still uses K&R style function prototypes, which are deprecated as of gcc 13.2 (in Ubuntu 24.04).
# Suppress the warning here for now.
```